tests/test_library/src/otherton.cpp

```diff
@@ -0,0 +1,9 @@
+#include <stdexcept>
+
+#include "simpleton.hpp"
+
+void noop(const HighFive::Object& obj) {
+    if (!obj.isValid()) {
```

Comment:
For completeness we could create a Boost array (ensure that the brought dependencies linking works)

Reply:
Should be a follow up, maybe?

Reply:
@matz-e maybe there was a little confusion, why not rename this test case to be a little more explicit in what it is testing?

Reply:
@ohm314 good point, although I'm a little stuck on a better name. `test_dependent_library` maybe? I can also add a note for all the headers…

Reply:
yeap, that would be not bad I think!